handler/configure_monitoring_triggered_event_handler_test.go

```diff
@@ -29,7 +29,7 @@ func TestHandleConfigureMonitoringTriggeredEvent(t *testing.T) {
 	//Building a mock resource service server
 	resourceServiceServer, err := buildMockResourceServiceServer(sliFilePath, shipyardFilePath, sloFilePath, remediationFilePath)
 	if err != nil {
-		t.Errorf("Error reading sli file : %s", err.Error())
+		t.Errorf("Error reading sli file : %v", err)
```

Comment:
You can use `t.Fatalf` instead of `t.Errorf` and remove `t.Fail` and have the same behavior